Makefile

```diff
@@ -14,7 +14,7 @@ build:
 	./build
 
 .PHONY: install
-install: build
+install: bin/ore bin/kola bin/plume bin/kolet
```

Comment:
OK by me, though at this point we might consider extracting a
`COMPONENTS := kola plume kolet`
variable and using it in multiple places.

Reply:
Yeah makes sense. Let's do that as a follow-up!